cmd/buildutil/runner.go

```diff
@@ -276,10 +278,15 @@ func (r *Runner) RunArtifacts(ctx context.Context, cmd *cobra.Command, args []st
 		case "deb":
 			version, release := r.Info.Version.StringRelease()
 
-			bindir := "/usr/share/bin"
-			files := map[string]string{}
+			bindir := "/usr/bin"
```

Comment:
Why?

Reply:
Because literally nothing goes in this directory, it wasn't even existing on my machine. Self-compiled stuff is in `/usr/local/bin`, packages go in `/usr/bin`

Reply:
Fair enough. I was sure I had this in my `$PATH`, but it is not there.